### PR TITLE
feat(github-actions): Create a Release workflow with GHA

### PR DESCRIPTION
**Pull request by NisanthanNanthakumar**
## Objective:
There is a new release system that uses Github Actions to create the release artifacts. So this PR is to implement this new workflow using [action-prepare-release](https://github.com/getsentry/action-prepare-release)

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      version:
+        description: Version to release
+        required: true
+      force:
+        description: Force a release even when there are release-blockers (optional)
+        required: false
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    name: "Release a new version"
```
Comment (scefali) [inline .github/workflows/release.yml:14]: @NisanthanNanthakumar nit: should we use string quotations here?

**Review by scefali — APPROVED** [review]

**Review by BYK — CHANGES_REQUESTED** [review]
So, I updated the [README over at getsentry/publish](https://github.com/getsentry/publish#usage) which details the implementation process. We need two more steps:

1. Add a proper `.craft.yml` file
2. Add @getsentry/releases as an admin to this repo

File: .github/workflows/release.yml (hunk 1 above)
Comment (NisanthanNanthakumar) [reply .github/workflows/release.yml:14]: @scefali Did you mean single quotes? Prettier converts it to double quotes.

**Comment by jan-auer** [comment]
> Add a proper .craft.yml file

To clarify this: The current [`.craft.yml`](https://github.com/getsentry/sentry-netlify-build-plugin/blob/91d932960f4e1c1c8ba4648cb837000f590b06a0/.craft.yml) uses a set of defaults that have to be updated. Please add:

```
minVersion: "0.14.0"

statusProvider:
  name: github
artifactProvider:
  name: github
```

**Comment by NisanthanNanthakumar** [comment]
> 2. Add @getsentry/releases as an admin to this repo

`@getsentry/releases` is now admin for `getsentry/sentry-netlify-build-plugin`

File: .craft.yml [hunk 2]
```diff
@@ -1,9 +1,14 @@
 ---
+minVersion: "0.14.0"
 github:
   owner: getsentry
   repo: sentry-netlify-build-plugin
 changelogPolicy: simple
```
Comment (BYK) [inline .craft.yml:6]: You may also consider `auto`: https://github.com/getsentry/craft#changelog-policies

**Review by BYK — APPROVED** [review]

**Pull request merged**